.firebase/hosting.ZGlzdA.cache

```diff
@@ -0,0 +1,46 @@
+favicon.ico,1618947961559,a6a3b41bac8b91c62ccbea30a56cc89ccf8059442e8f58255bebe47c84a51c66
```

Comment:
detele/gitignore?

Reply:
yeah... I wasn't sure... other projects have it so that's why I left it. But if you think we don't need it, I'm happy to gitignore it

Reply:
The advice I'm seeing says to gitignore it (https://stackoverflow.com/questions/52130772/whats-the-purpose-of-firebase-hosting-alphanum-cache https://circleci.com/blog/automatically-deploy-a-gatsby-site-to-firebase-hosting/ ) but I'm also not finding a ton of advice out there

Reply:
I'll ignore it!